roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -2252,31 +2252,21 @@ RooDataSet *RooAbsPdf::generate(const RooArgSet &whatVars, const RooDataSet& pro
 
```

Comment:
i know it's unrelated, but the docstring seems wrong, the `nGen` parameter has no effect (I assume it's the second unused parameter)

Reply:
Thanks for spotting that!